cypress/integration/inventory/item.status.date.updates.spec.js

```diff
@@ -0,0 +1,317 @@
+          user.userId, effectiveLocationServicePoint.body.id);
+        cy.login(userProperties.username, userProperties.password)
+          .then(() => {
+            NewLocations.createViaApi(NewLocations.getDefaultLocation(effectiveLocationServicePointId))
+              .then((location) => {
+                effectiveLocation = location;
+                Orders.createOrderWithOrderLineViaApi(
+                  NewOrder.getDefaultOrder(),
+                  BasicOrderLine.getDefaultOrderLine(itemQuantity, instanceTitle, location.id)
+                )
+                  .then(order => {
+                    orderNumber = order;
+                  });
+              });
+          });
+
+        cy.getUsers({ limit: 1, query: `"personal.lastName"="${user.username}" and "active"="true"` })
+          .then((users) => {
+            userBarcode = users[0].barcode;
+          });
+      });
+
+    cy.createTempUser([
+      permissions.checkoutAll.gui,
+      permissions.requestsAll.gui,
+    ])
+      .then(userProperties => {
+        userForDeliveryRequest = userProperties;
+        cy.getUsers({ limit: 1, query: `"username"="${userForDeliveryRequest.username}"` })
+          .then((users) => {
+            userForDeliveryRequest.barcode = users[0].barcode;
+            userRequestPreferences.userId = users[0].id;
+            cy.getAddressTypesApi({ limit: 1 }).then(addressTypes => {
+              userRequestPreferences.defaultDeliveryAddressTypeId = addressTypes[0].id;
+            })
+              .then(() => {
+                UpdateUser.updateUserAddress(users[0], [{ city: 'New York',
+                  addressTypeId: userRequestPreferences.defaultDeliveryAddressTypeId,
+                  countryId: 'US' }]);
+                cy.createUserRequestPreferencesApi(userRequestPreferences);
+              });
+          });
+      });
+  });
+
+  afterEach(() => {
+    InventoryInstances.deleteInstanceViaApi(userItemBarcode);
+    cy.getOrdersApi({ limit: 1, query: `"poNumber"=="${orderNumber}"` })
+      .then(order => {
+        cy.deleteOrderApi(order[0].id);
+      });
+    cy.getOrganizationApi()
+      .then(organization => {
+        cy.deleteOrganizationApi(organization[0].id);
+      });
+    cy.deleteServicePoint(notEffectiveLocationServicePoint.body.id);
+    SwitchServicePoint.changeServicePointPreference(user.username);
+    Requests.getRequestApi({ limit: 1, query: `"item.barcode"=="${userItemBarcode}"` })
+      .then(requests => {
+        requests.forEach(request => {
+          Requests.deleteRequestApi(request.id);
+        });
+      });
+    cy.deleteUser(user.userId);
+  });
+
+  it('C9200 Item status date updates', { tags: [TestTypes.smoke] }, () => {
+    const caption = `autotest_caption_${getRandomPostfix()}`;
+    const numberOfPieces = '3';
+
+    cy.visit(TopMenu.ordersPath);
+    Orders.searchByParameter('PO number', orderNumber);
```

Comment:
add  wrappers into the test. See repeated steps

Reply:
Done